admin-data-explorer-play-server/src/test/scala/controllers/AdminControllerSpec.scala

```diff
@@ -255,6 +602,27 @@ class AdminControllerSpec extends PlayPostgresSpec {
       emailResponse must be(email)
     }
 
+    "update a new row for all tables" in withApiClient { client =>
+      val tables = List(serialSettings, bigSerialSettings)
+      for (table <- tables) {
+        val request = AdminCreateTable.Request(
+          Map("name" -> "wiringbits")
+        )
+        client.createItem(table.tableName, request).futureValue
+        val response = client.getTableMetadata(table.tableName, List("name", "ASC"), List(0, 9), "{}").futureValue
+        val id = response.headOption.value.find(_._1 == "id").value._2
+
+        val name = "wiringbitsbitsbits"
+        val updateRequest = Map("name" -> name)
+        val updateResponse = client.updateItem(table.tableName, id, updateRequest).futureValue
+
+        val newResponse = client.viewItem(table.tableName, id).futureValue
+        val nameResponse = newResponse.find(_._1 == "name").value._2
```

Comment:
An alternative can be `val (_, nameResponse) = newResponse.find(_._1 == "name")`